### ISSUE TITLE
[Low Priority] Add Ruby RSpec file to aerial.nvim

### ISSUE THREAD

**Issue by olimorris** [stevearc/aerial.nvim#40]
**Language**: Ruby

As per discussions on #35, adding function calls for a Ruby RSpec file.

Example language file (taken from [here](https://github.com/nicenine98/square-ruby-/blob/main/spec/models/v1_modifier_option_spec.rb))

```ruby
require 'spec_helper'
require 'json'
require 'date'

# Unit tests for SquareConnect::V1ModifierOption
# Automatically generated by swagger-codegen (github.com/swagger-api/swagger-codegen)
# Please update as you see appropriate
describe 'V1ModifierOption' do
  before do
    # run before each test
    @instance = SquareConnect::V1ModifierOption.new
  end

  after do
    # run after each test
  end

  describe 'test an instance of V1ModifierOption' do
    it 'should create an instact of V1ModifierOption' do
      expect(@instance).to be_instance_of(SquareConnect::V1ModifierOption)
    end
  end
  describe 'test attribute "id"' do
    it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
    end
  end

  describe 'test attribute "name"' do
    it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
    end
  end

  describe 'test attribute "price_money"' do
    it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
    end
  end

  describe 'test attribute "on_by_default"' do
    it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
    end
  end

  describe 'test attribute "ordinal"' do
    it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
    end
  end

  describe 'test attribute "modifier_list_id"' do
    it 'should work' do
       # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
    end
  end

end
```

## Thoughts on implementation

Appreciate that implementing function calls could lead to `aerial.nvim` becoming bloated from a UI perspective. Is there anyway to load function calls for specific file types or names like `_spec.rb`?

**Comment by stevearc** [comment]
Implemented via treesitter backend in 3e62af55fbe66e48c5b02e8ea4f0a067c2da47ae. Give it a try!

**Comment by olimorris** [comment]
Amazing. Our damn `RSpec` files get so big that being able to use aerial is a huge win. Many thanks!
